File: backend/metier_generator/templates/threed/hooks/custom_event/custom_event_hooks.py
"""
Hooks custom event threed (Python)
"""


def before_custom_event(event, data):
    # Logique avant événement custom
    return {"event": event, "data": data}


def after_custom_event(event, data):
    # Logique après événement custom
    return {"event": event, "data": data}

File: backend/metier_generator/templates/threed/hooks/custom_event/custom_event_hooks.js
// Hooks custom event threed (Node.js)
function beforeCustomEvent(event, data) {
  // Logique avant événement custom
  return { event, data };
}
function afterCustomEvent(event, data) {
  // Logique après événement custom
  return { event, data };
}
module.exports = { beforeCustomEvent, afterCustomEvent };
